src/jit/emitxarch.cpp

```diff
@@ -4821,6 +4821,13 @@ void emitter::emitIns_S_R(instruction ins, emitAttr attr, regNumber ireg, int va
     UNATIVE_OFFSET sz  = emitInsSizeSV(insCodeMR(ins), varx, offs);
     insFormat      fmt = emitInsModeFormat(ins, IF_SRD_RRD);
 
+    assert(attr != EA_1BYTE);
+#ifdef _TARGET_X86_
+    if (attr == EA_1BYTE)
```

Comment:
It is a bit weird that we assert that `attr` is not `EA_1BYTE` above and then immediately handle the case where it might be with another assertion. Can you add a comment here?